javascripts/application.js

```diff
@@ -110,6 +110,14 @@ $(document).ready(function() {
                       $(this).text(
                           text == "Show Me >" ? "Hide Image" : "Show Me >");
   }));
+
+  // Add anchorJS anchors; see https://github.com/18F/content-guide/issues/149
+  anchors.options.placement = 'right';
+  anchors.add();
+  anchors.remove(".logo");
+  anchors.remove("#title-18f-content-guide");
+  anchors.remove("ol li h1");
```

Comment:
Oh geez, that is too bad about kramdown! Maybe add  comment somewhere mentioning this? Up to you really.